src/WormholeEndpointStandalone.sol

```diff
@@ -14,6 +14,12 @@ contract WormholeEndpointStandalone is WormholeEndpoint, EndpointStandalone, Own
         address wormholeRelayerAddr
     ) EndpointStandalone(manager) WormholeEndpoint(wormholeCoreBridge, wormholeRelayerAddr) {}
 
+    /// @notice This function is used to pause the endpoint
+    /// Only the implementor (deployer) can call this function
```

Comment:
This comment won't be true soon so probably worth removing/updating

Reply:
true